src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
         and we will work with you and the appropriate development team to resolve it.
         Any fixes we identify will be delivered as part of the next AdoptOpenJDK release.
       </p>
+	  
+		<h3 id="openjdk8_platforms">OpenJDK 8</h3>
+		<p>Where available, OpenJDK 8 binaries are supported on the minimum operating system levels shown in the following tables:</p>
+
+		<table border="1">
+			<thead>
+				<tr>
+					<th>Key</th>
+					<th>Value</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>HotSpot variant</td>
```

Comment:
Can you change the word variant to VM everywhere (we need to do this on other pages as well, but let's start here as a good example to follow).

Reply:
Done